R/derive_advs_params.R

```diff
@@ -255,6 +255,15 @@ compute_map <- function(diabp, sysbp, hr = NULL) {
 #'
 #'   Permitted Values: character value
 #'
+#' @param constant_height HEIGHT parameter is constant
+#'
+#'   If the HEIGHT value is considered constant (e.g. measured once at screening),
+#'   the values of BSA will be calculated for each WEIGHT value.
+#'
+#'   Permitted Values: logical scalar
+#'
+#'   Default Values: FALSE
```

Comment:
I think the convention now in {admiral} is to not specify the default value in the roxygen header.